src/toil/jobStores/abstractJobStore.py

```diff
@@ -903,6 +904,15 @@ def writeFileStream(self, jobStoreID=None, cleanup=False, basename=None):
                file basename so that when searching the job store with a query
                matching that basename, the file will be detected.
 
+        :param str mode: an optional string that specifies the mode in which the file is opened.
+                It defaults to 'b' which means open for reading in binary mode.
+
+        :param str encoding: the name of the encoding used to decode or encode the file.
+                This should only be used in text mode.
+
+        :param str errors: an optional string that specifies how encoding and decoding errors
+                are to be handled. This cannot be used in binary mode.
```

Comment:
Similarly, this should explain what values are acceptable, or where the list of acceptable values is. Are all the ones for `str.encode()` supported? And what is the default?